src/app/pages/FrontPage/index.jsx

```diff
@@ -132,8 +134,29 @@ const FrontPage = ({ pageData, mostReadEndpointOverride }) => {
     />
   );
 
+  const CanonicalAdBootstrapJs = () => (
```

Comment:
I think this `CanonicalAdBootstrapJs` should be its own file here:
`src/app/containers/Ad/Canonical/BootstrapJs.jsx`.
and imported in as 
```import CanonicalAdBootstrapJs from '#app/containers/Ad/Canonical/BootstrapJs.jsx';```
Why a separate file? Easy to unit test.
Why under`/Ad/Canonical`? Because that's where all the other canonical ads components are. It makes sense to group these together.

Reply:
Done